### ISSUE TITLE
Save button in setting does nothing

### ISSUE THREAD

**Issue by Savanta** [jokob-sk/NetAlertX#636]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues and I checked the docs https://github.com/jokob-sk/NetAlertX/tree/main/docs

### Current Behavior

Save button in settings not working, manualny editing config file works

### Expected Behavior

Saving settings give effect

### Steps To Reproduce

1. deploy via docker compose
2. go to setting
3. add new subnet
4. click save

### app.conf

```python
#-----------------AUTOGENERATED FILE-----------------#
#                                                    #
#         Generated:  2022-12-30_22-19-40            #
#                                                    #
#   Config file for the LAN intruder detection app:  #
#      https://github.com/jokob-sk/NetAlertX          #
#                                                    #
#-----------------AUTOGENERATED FILE-----------------#

# 🔺 Use the Settings UI - only edit when necessary 🔺


# General
#---------------------------
# Scan using interface eth0
# SCAN_SUBNETS    = ['192.168.1.0/24 --interface=eth0']
#
# Scan multiple interfaces (eth1 and eth0):
# SCAN_SUBNETS    = [ '192.168.1.0/24 --interface=eth1', '192.168.1.0/24 --interface=eth0' ]

SCAN_SUBNETS=['192.168.1.0/24 --interface=eth1', '10.0.0.0/24 --interface=eth0']

TIMEZONE='Europe/Warsaw'
PIALERT_WEB_PROTECTION=False
PIALERT_WEB_PASSWORD='8d969eef6ecad3c29a3a629280e686cf0c3f5d5a86aff3ca12020c923adc6c92'
DAYS_TO_KEEP_EVENTS=90
# Used for generating links in emails. Make sure not to add a trailing slash!
REPORT_DASHBOARD_URL='https://pialert.think.fox'


# Email
#---------------------------
SMTP_RUN='disabled'  # use 'on_notification' to enable
SMTP_SERVER='smtp.gmail.com'
SMTP_PORT=587
SMTP_REPORT_TO='user@gmail.com'
SMTP_REPORT_FROM='NetAlertX <user@gmail.com>'
SMTP_SKIP_LOGIN=False
SMTP_USER='user@gmail.com'
SMTP_PASS='password'
SMTP_SKIP_TLS=False


# Webhooks
#---------------------------
WEBHOOK_RUN='disabled'  # use 'on_notification' to enable
WEBHOOK_URL='http://n8n.local:5555/webhook-test/aaaaaaaa-aaaa-aaaa-aaaaa-aaaaaaaaaaaa'
WEBHOOK_PAYLOAD='json'                 # webhook payload data format for the "body > attachements > text" attribute 
                                       # in https://github.com/jokob-sk/NetAlertX/blob/main/docs/webhook_json_sample.json 
                                       #   supported values: 'json', 'html' or 'text'
                                       #   e.g.: for discord use 'html'
WEBHOOK_REQUEST_METHOD='GET'


# Apprise
#---------------------------
APPRISE_RUN='disabled'  # use 'on_notification' to enable
APPRISE_HOST='http://localhost:8000/notify'
APPRISE_URL='mailto://smtp-relay.sendinblue.com:587?from=user@gmail.com&name=apprise&user=user@gmail.com&pass=password&to=user@gmail.com'


# NTFY
#---------------------------
NTFY_RUN='disabled'  # use 'on_notification' to enable
NTFY_HOST='https://ntfy.sh'
NTFY_TOPIC='replace_my_secure_topicname_91h889f28'
NTFY_USER='user'
NTFY_PASSWORD='passw0rd'


# PUSHSAFER
#---------------------------
PUSHSAFER_RUN='disabled'  # use 'on_notification' to enable
PUSHSAFER_TOKEN='ApiKey'


# MQTT
#---------------------------
MQTT_RUN='disabled'  # use 'on_notification' to enable
MQTT_BROKER='192.168.1.2'
MQTT_PORT=1883
MQTT_USER='mqtt'
MQTT_PASSWORD='passw0rd'
MQTT_QOS=0
MQTT_DELAY_SEC=2


#-------------------IMPORTANT INFO-------------------#
#   This file is ingested by a python script, so if  #
#        modified it needs to use python syntax      #
#-------------------IMPORTANT INFO-------------------#
```

### docker-compose.yml

```python
services:
  netalertx:
    container_name: netalertx
    # use the below line if you want to test the latest dev image
    # image: "jokobsk/netalertx-dev:latest" 
    image: jokobsk/netalertx:latest
    network_mode: host
    restart: unless-stopped
    volumes:
      - ./config:/app/config
      - ./db:/app/db
      - ./logs:/app/front/log
      - /etc/pihole/pihole-FTL.db:/etc/pihole/pihole-FTL.db
      - /etc/pihole/dhcp.leases:/etc/pihole/dhcp.leases
    environment:
      - TZ=Europe/Warsaw
      - PORT=20211
networks: {}
```

### What branch are you running?

Production

### app.log

15:45:54 [MAIN] Setting up ...
15:45:54 [conf.tz] Setting up ...
15:45:54 

15:45:54 The container restarted (started). If this is unexpected check https://bit.ly/NetAlertX_debug for troubleshooting tips.
15:45:54 

15:45:54 Permissions check (All should be True)
15:45:54 ------------------------------------------------
15:45:54   /config/app.conf |  READ  | True
15:45:54   /config/app.conf |  WRITE | True
15:45:54   /db/app.db       |  READ  | True
15:45:54   /db/app.db       |  WRITE | True
15:45:54 ------------------------------------------------
15:45:54 [Setup] Attempting to fix permissions.
15:45:54 [Setup] Attempting to fix permissions.
15:45:54 [Database] Opening DB
15:45:54 [upgradeDB] Adding dev_Network_Node_MAC_ADDR to the Devices table
15:45:54 [upgradeDB] Adding dev_Network_Node_port to the Devices table
15:45:54 [upgradeDB] Adding dev_Icon to the Devices table
15:45:54 [upgradeDB] Re-creating Settings table
15:45:54 [upgradeDB] Re-creating Pholus_Scan table
15:45:54 [upgradeDB] Re-creating Parameters table
15:45:55 [Version check] Running the latest version.
15:45:55 [Config] reading config file
15:45:55 [Config] Plugins: Number of dynamically loaded plugins: 29
15:45:55 [Plugin utils] ---------------------------------------------
15:45:55 [Plugin utils] display_name: PiHole (Device sync)
15:45:55 [Plugin utils] description: This plugin syncs devices from the PiHole database
15:45:55 [Plugin utils] ---------------------------------------------
15:45:55 [Plugin utils] display_name: Set password
15:45:55 [Plugin utils] description: A simple plugin to set the web ui password on app start.
15:45:55 [Plugin utils] ---------------------------------------------
15:45:55 [Plugin utils] display_name: New Devices
15:45:55 [Plugin utils] description: The template used for new devices.
15:45:55 [Plugin utils] ---------------------------------------------
15:45:55 [Plugin utils] display_name: CSV backup
15:45:55 [Plugin utils] description: A plugin to auto-generate devices.csv backups.
15:45:56 [Plugin utils] ---------------------------------------------
15:45:56 [Plugin utils] display_name: Pushover publisher
15:45:56 [Plugin utils] description: A plugin to publish a notification via the pushover.net
15:45:56 [Plugin utils] ---------------------------------------------
15:45:56 [Plugin utils] display_name: UniFi import
15:45:56 [Plugin utils] description: This plugin is used to import devices from an UNIFI controller.
15:45:56 [Plugin utils] ---------------------------------------------
15:45:56 [Plugin utils] display_name: MQTT publisher
15:45:56 [Plugin utils] description: A plugin to publish a notification via the Apprise gateway.
15:45:56 [Plugin utils] ---------------------------------------------
15:45:56 [Plugin utils] display_name: Internet-Check
15:45:56 [Plugin utils] description: A plugin to check your internet connectivity and IP.
15:45:56 [Plugin utils] ---------------------------------------------
15:45:56 [Plugin utils] display_name: Internet speedtest
15:45:56 [Plugin utils] description: A plugin to perform a scheduled internet speedtest.
15:45:56 [Plugin utils] ---------------------------------------------
15:45:56 [Plugin utils] display_name: Email publisher (SMTP)
15:45:56 [Plugin utils] description: A plugin to publish a notification via Email (SMTP) gateway.
15:45:56 [Plugin utils] ---------------------------------------------
15:45:56 [Plugin utils] display_name: Website monitor
15:45:56 [Plugin utils] description: This plugin is to monitor status changes of services or websites.
15:45:56 [Plugin utils] ---------------------------------------------
15:45:56 [Plugin utils] display_name: Apprise publisher
15:45:56 [Plugin utils] description: A plugin to publish a notification via the Apprise gateway.
15:45:56 [Plugin utils] ---------------------------------------------
15:45:57 [Plugin utils] display_name: Arp-Scan (Network scan)
15:45:57 [Plugin utils] description: This plugin is to execute an arp-scan on the local network
15:45:57 [Plugin utils] ---------------------------------------------
15:45:57 [Plugin utils] display_name: Rogue DHCP
15:45:57 [Plugin utils] description: This plugin is to use NMAP to monitor for rogue DHCP servers.
15:45:57 [Plugin utils] ---------------------------------------------
15:45:57 [Plugin utils] display_name: DDNS update
15:45:57 [Plugin utils] description: A plugin update the DDNS record.
15:45:57 [Plugin utils] ---------------------------------------------
15:45:57 [Plugin utils] display_name: Workflows
15:45:57 [Plugin utils] description: A plugin to adjust behavior of workflows.
15:45:57 [Plugin utils] ---------------------------------------------
15:45:57 [Plugin utils] display_name: Un-Discoverable Devices
15:45:57 [Plugin utils] description: This plugin is to import undiscoverable devices from a file.
15:45:57 [Plugin utils] ---------------------------------------------
15:45:57 [Plugin utils] display_name: NTFY publisher
15:45:57 [Plugin utils] description: A plugin to publish a notification via the NTFY gateway.
15:45:57 [Plugin utils] ---------------------------------------------
15:45:57 [Plugin utils] display_name: Pholus (Name discovery)
15:45:57 [Plugin utils] description: This plugin is to execute a Pholus (name discovery) on the local network
15:45:57 [Plugin utils] ---------------------------------------------
15:45:57 [Plugin utils] display_name: Maintenance
15:45:57 [Plugin utils] description: A plugin for maintenance tasks.
15:45:57 [Plugin utils] ---------------------------------------------
15:45:57 [Plugin utils] display_name: NSLOOKUP (Name discovery)
15:45:57 [Plugin utils] description: A plugin to discover device names.
15:45:57 [Plugin utils] ---------------------------------------------
15:45:57 [Plugin utils] display_name: SNMP discovery
15:45:57 [Plugin utils] description: This plugin is used to discover devices via the arp table(s) of a RFC1213 compliant router or switch.
15:45:57 [Plugin utils] ---------------------------------------------
15:45:57 [Plugin utils] display_name: DHCP Leases (Device import)
15:45:57 [Plugin utils] description: This plugin is to import devices from dhcp.leases files.
15:45:57 [Plugin utils] ---------------------------------------------
15:45:57 [Plugin utils] display_name: Services & Ports (NMAP)
15:45:57 [Plugin utils] description: This plugin shows all services discovered by NMAP scans.
15:45:57 [Plugin utils] ---------------------------------------------
15:45:57 [Plugin utils] display_name: Vendor update
15:45:57 [Plugin utils] description: A plugin to schedule vendor database updates for mac based vendor resolution.
15:45:57 [Plugin utils] ---------------------------------------------
15:45:57 [Plugin utils] display_name: Webhook publisher
15:45:57 [Plugin utils] description: A plugin to publish a notification via Webhooks.
15:45:57 [Plugin utils] ---------------------------------------------
15:45:57 [Plugin utils] display_name: Pushsafer publisher
15:45:57 [Plugin utils] description: A plugin to publish a notification via the Pushsafer gateway.
15:45:57 [Plugin utils] ---------------------------------------------
15:45:57 [Plugin utils] display_name: DB cleanup
15:45:57 [Plugin utils] description: A plugin to schedule database cleanup & upkeep tasks.
15:45:57 [Plugin utils] ---------------------------------------------
15:45:57 [Plugin utils] display_name: Notification Processing
15:45:57 [Plugin utils] description: A plugin to for advanced notification processing.
15:45:58 [API] Updating table_settings.json file in /front/api
15:45:58 [Config] Imported new config
15:45:58 [API] Updating table_appevents.json file in /front/api
15:45:58 [API] Updating table_devices.json file in /front/api
15:45:58 [API] Updating table_events_pending_alert.json file in /front/api
15:45:58 [API] Updating table_plugins_events.json file in /front/api
15:45:58 [API] Updating table_plugins_history.json file in /front/api
15:45:58 [API] Updating table_plugins_objects.json file in /front/api
15:45:58 [API] Updating table_plugins_language_strings.json file in /front/api
15:45:58 [Database] - SQL ERROR: no such table: Notifications
15:45:58 [API] Updating table_notifications.json file in /front/api
15:45:58 [API] Updating table_custom_endpoint.json file in /front/api
15:45:58 [Plugin utils] ---------------------------------------------
15:45:58 [Plugin utils] display_name: NSLOOKUP (Name discovery)
15:45:58 [Plugins] Executing: python3 /app/front/plugins/nslookup_scan/nslookup.py
15:45:58 [NSLOOKUP] In script
15:45:58 [Database] Opening DB
15:45:58 [NSLOOKUP] Unknown devices count: 0
15:45:58 [NSLOOKUP] Script finished
15:45:58 [Plugins] No output received from the plugin NSLOOKUP - enable LOG_LEVEL=debug and check logs
15:45:58 [Notification] Check if something to report
15:45:58 [Notification] Included sections: ['new_devices', 'down_devices', 'events']
15:45:58 [Notification] No changes to report
15:45:58 [MAIN] Process: Wait
15:46:03 [API] Updating table_notifications.json file in /front/api
15:46:59 [Plugin utils] ---------------------------------------------
15:46:59 [Plugin utils] display_name: NSLOOKUP (Name discovery)
15:46:59 [Plugins] Executing: python3 /app/front/plugins/nslookup_scan/nslookup.py
15:46:59 [NSLOOKUP] In script
15:46:59 [Database] Opening DB
15:46:59 [NSLOOKUP] Unknown devices count: 0
15:46:59 [NSLOOKUP] Script finished
15:46:59 [Plugins] No output received from the plugin NSLOOKUP - enable LOG_LEVEL=debug and check logs
15:46:59 [Notification] Check if something to report
15:46:59 [Notification] Included sections: ['new_devices', 'down_devices', 'events']
15:46:59 [Notification] No changes to report
15:46:59 [MAIN] Process: Wait
15:48:04 [Plugin utils] ---------------------------------------------
15:48:04 [Plugin utils] display_name: NSLOOKUP (Name discovery)
15:48:04 [Plugins] Executing: python3 /app/front/plugins/nslookup_scan/nslookup.py
15:48:05 [NSLOOKUP] In script
15:48:05 [Database] Opening DB
15:48:05 [NSLOOKUP] Unknown devices count: 0
15:48:05 [NSLOOKUP] Script finished
15:48:05 [Plugins] No output received from the plugin NSLOOKUP - enable LOG_LEVEL=debug and check logs
15:48:05 [Notification] Check if something to report
15:48:05 [Notification] Included sections: ['new_devices', 'down_devices', 'events']
15:48:05 [Notification] No changes to report
15:48:05 [MAIN] Process: Wait
15:49:05 [Plugin utils] ---------------------------------------------
15:49:05 [Plugin utils] display_name: NSLOOKUP (Name discovery)
15:49:05 [Plugins] Executing: python3 /app/front/plugins/nslookup_scan/nslookup.py
15:49:05 [NSLOOKUP] In script
15:49:05 [Database] Opening DB
15:49:05 [NSLOOKUP] Unknown devices count: 0
15:49:05 [NSLOOKUP] Script finished
15:49:05 [Plugins] No output received from the plugin NSLOOKUP - enable LOG_LEVEL=debug and check logs
15:49:05 [Notification] Check if something to report
15:49:05 [Notification] Included sections: ['new_devices', 'down_devices', 'events']
15:49:05 [Notification] No changes to report
15:49:05 [MAIN] Process: Wait
15:50:06 [Plugin utils] ---------------------------------------------
15:50:06 [Plugin utils] display_name: Internet-Check
15:50:06 [Plugins] Executing: python3 /app/front/plugins/internet_ip/script.py prev_ip={prev_ip} INTRNT_DIG_GET_IP_ARG={INTRNT_DIG_GET_IP_ARG}
15:50:06 [INTRNT] In script
15:50:06 [INTRNT] INTRNT_DIG_GET_IP_ARG: -4 myip.opendns.com @resolver1.opendns.com
15:50:06 [INTRNT] - Retrieving Internet IP
15:50:06 [INTRNT]  DIG result : 95.160.111.16

15:50:06 [INTRNT]  Current internet_IP : 95.160.111.16
15:50:06 [INTRNT]          previous_IP : 0.0.0.0
15:50:06 [INTRNT] Finished 
15:50:06 [Plugins] SUCCESS, received 1 entries
15:50:06 [API] Updating table_appevents.json file in /front/api
15:50:06 [API] Updating table_plugins_events.json file in /front/api
15:50:06 [API] Updating table_plugins_history.json file in /front/api
15:50:06 [API] Updating table_plugins_objects.json file in /front/api
15:50:06 [Plugin utils] ---------------------------------------------
15:50:06 [Plugin utils] display_name: Arp-Scan (Network scan)
15:50:06 [Plugins] Executing: python3 /app/front/plugins/arp_scan/script.py userSubnets={subnets}
15:50:06 [ARP Scan] In script 
15:50:06 [ARP Scan] values.userSubnets: ["userSubnets=b'MTkyLjE2OC4xLjAvMjQgLS1pbnRlcmZhY2U9ZXRoMQ=='"]
15:50:06 [ARP Scan] userSubnetsParamBase64: 'MTkyLjE2OC4xLjAvMjQgLS1pbnRlcmZhY2U9ZXRoMQ=='
15:50:06 [ARP Scan] userSubnetsParam: 192.168.1.0/24 --interface=eth1
15:50:06 [Database] Opening DB
15:50:06 [ARP Scan] arpscan_output: 
15:50:06 [ARP Scan] Found: Devices without duplicates 0
15:50:06 Devices List len:0
15:50:06 Devices List:[]
15:50:06 [Plugins] No output received from the plugin ARPSCAN - enable LOG_LEVEL=debug and check logs
15:50:06 [Process Scan]  Processing scan results
15:50:06 [Process Scan] Print Stats
15:50:06 [Scan Stats] Devices Detected.......: 2
15:50:06 [Scan Stats] New Devices............: 1
15:50:06 [Scan Stats] Down Alerts............: 0
15:50:06 [Scan Stats] New Down Alerts........: 0
15:50:06 [Scan Stats] New Connections........: 0
15:50:06 [Scan Stats] Disconnections.........: 0
15:50:06 [Scan Stats] IP Changes.............: 1
15:50:06 [Scan Stats] Scan Method Statistics:
15:50:06     INTRNT: 1
15:50:06     local_MAC: 1
15:50:06 [Process Scan] Stats end
15:50:06 [Process Scan] Sessions Events (connect / discconnect)
15:50:06 [Process Scan] Creating new devices
15:50:06 [Process Scan] Updating Devices Info
15:50:06 [Process Scan] Voiding false (ghost) disconnections
15:50:06 [Process Scan] Pairing session events (connection / disconnection) 
15:50:06 [Process Scan] Creating sessions snapshot
15:50:06 [Process Scan] Inserting scan results into Online_History
15:50:06 [Process Scan] Skipping repeated notifications
15:50:06 [Skip Repeated Notifications] Skip Repeated
15:50:06 [Plugin utils] ---------------------------------------------
15:50:06 [Plugin utils] display_name: NSLOOKUP (Name discovery)
15:50:06 [Plugins] Executing: python3 /app/front/plugins/nslookup_scan/nslookup.py
15:50:06 [NSLOOKUP] In script
15:50:06 [Database] Opening DB
15:50:06 [NSLOOKUP] Unknown devices count: 1
15:50:06 [NSLOOKUP] Domain Name: thinkfox.lan.
15:50:06 [NSLOOKUP] Script finished
15:50:06 [Plugins] SUCCESS, received 1 entries
15:50:06 [API] Updating table_appevents.json file in /front/api
15:50:06 [API] Updating table_plugins_history.json file in /front/api
15:50:06 [API] Updating table_plugins_objects.json file in /front/api
15:50:06 [Update Device Name] Trying to resolve devices without name. Unknown devices count: 1
15:50:06 [Update Device Name] Pholus entries from prev scans: 0
15:50:07 [Update Device Name] Names Found (DiG/NSLOOKUP/Pholus): 1 (1/0/0)
15:50:07 [Update Device Name] Names Not Found         : 0
15:50:07 [Plugin utils] ---------------------------------------------
15:50:07 [Plugin utils] display_name: Pholus (Name discovery)
15:50:07 [Plugins] Executing: python3 /app/front/plugins/pholus_scan/script.py userSubnets={subnets} timeoutSec={timeout}
15:50:07 [PHOLUS] In script
15:50:07 [PHOLUS] Subnets: ["userSubnets=b'MTkyLjE2OC4xLjAvMjQgLS1pbnRlcmZhY2U9ZXRoMQ=='"]
15:50:07 [PHOLUS] len Subnets: 1
15:50:07 [PHOLUS] 'MTkyLjE2OC4xLjAvMjQgLS1pbnRlcmZhY2U9ZXRoMQ=='
15:50:07 [PHOLUS] 300
15:50:07 [PHOLUS] userSubnetsParam 192.168.1.0/24 --interface=eth1 
15:50:07 [PHOLUS] 300.0 
15:50:07 [PHOLUS] Scan: Pholus for 300.0s (5.0min)
15:50:07 [PHOLUS] Pholus scan on [interface] eth1 [mask] 192.168.1.0/24
15:50:07 [PHOLUS] Scan: Pholus SUCCESS
15:50:07 [PHOLUS] [] 
15:50:07 [PHOLUS] Pholus output number of entries:0
15:50:07 [PHOLUS] List:[]
15:50:07 [Plugins] No output received from the plugin PHOLUS - enable LOG_LEVEL=debug and check logs
15:50:07 [Notification] Check if something to report
15:50:07 [Notification] Included sections: ['new_devices', 'down_devices', 'events']
15:50:07 [Notification] Open text Template
15:50:07 [Notification] Open html Template
15:50:07 [Notification] Using template/app/front/report_templates/report_template.html
15:50:07 [Notification] New Devices sections done.
15:50:07 [Notification] Down Devices sections done.
15:50:07 [Notification] Events sections done.
15:50:07 [Notification] Plugins sections done.
15:50:07 [Send API] Updating notification_* files in /app/front/api/
15:50:07 [Notification] Udating API files
15:50:07 [Notification] Notifications changes: 1
15:50:07 [MAIN] Process: Wait
15:50:12 [API] Updating table_appevents.json file in /front/api
15:50:12 [API] Updating table_devices.json file in /front/api
15:50:12 [API] Updating table_plugins_events.json file in /front/api
15:50:12 [API] Updating table_notifications.json file in /front/api
15:51:08 [Plugin utils] ---------------------------------------------
15:51:08 [Plugin utils] display_name: NSLOOKUP (Name discovery)
15:51:08 [Plugins] Executing: python3 /app/front/plugins/nslookup_scan/nslookup.py
15:51:08 [NSLOOKUP] In script
15:51:08 [Database] Opening DB
15:51:08 [NSLOOKUP] Unknown devices count: 0
15:51:08 [NSLOOKUP] Script finished
15:51:08 [Plugins] No output received from the plugin NSLOOKUP - enable LOG_LEVEL=debug and check logs
15:51:08 [Notification] Check if something to report
15:51:08 [Notification] Included sections: ['new_devices', 'down_devices', 'events']
15:51:08 [Notification] No changes to report
15:51:08 [MAIN] Process: Wait
15:52:13 [Plugin utils] ---------------------------------------------
15:52:13 [Plugin utils] display_name: NSLOOKUP (Name discovery)
15:52:13 [Plugins] Executing: python3 /app/front/plugins/nslookup_scan/nslookup.py
15:52:13 [NSLOOKUP] In script
15:52:13 [Database] Opening DB
15:52:13 [NSLOOKUP] Unknown devices count: 0
15:52:13 [NSLOOKUP] Script finished
15:52:13 [Plugins] No output received from the plugin NSLOOKUP - enable LOG_LEVEL=debug and check logs
15:52:13 [Notification] Check if something to report
15:52:13 [Notification] Included sections: ['new_devices', 'down_devices', 'events']
15:52:13 [Notification] No changes to report
15:52:13 [MAIN] Process: Wait
15:53:14 [Plugin utils] ---------------------------------------------
15:53:14 [Plugin utils] display_name: NSLOOKUP (Name discovery)
15:53:14 [Plugins] Executing: python3 /app/front/plugins/nslookup_scan/nslookup.py
15:53:14 [NSLOOKUP] In script
15:53:14 [Database] Opening DB
15:53:14 [NSLOOKUP] Unknown devices count: 0
15:53:14 [NSLOOKUP] Script finished
15:53:14 [Plugins] No output received from the plugin NSLOOKUP - enable LOG_LEVEL=debug and check logs
15:53:14 [Notification] Check if something to report
15:53:14 [Notification] Included sections: ['new_devices', 'down_devices', 'events']
15:53:14 [Notification] No changes to report
15:53:14 [MAIN] Process: Wait
15:54:19 [Plugin utils] ---------------------------------------------
15:54:19 [Plugin utils] display_name: NSLOOKUP (Name discovery)
15:54:19 [Plugins] Executing: python3 /app/front/plugins/nslookup_scan/nslookup.py
15:54:20 [NSLOOKUP] In script
15:54:20 [Database] Opening DB
15:54:20 [NSLOOKUP] Unknown devices count: 0
15:54:20 [NSLOOKUP] Script finished
15:54:20 [Plugins] No output received from the plugin NSLOOKUP - enable LOG_LEVEL=debug and check logs
15:54:20 [Notification] Check if something to report
15:54:20 [Notification] Included sections: ['new_devices', 'down_devices', 'events']
15:54:20 [Notification] No changes to report
15:54:20 [MAIN] Process: Wait
15:55:20 [Plugin utils] ---------------------------------------------
15:55:20 [Plugin utils] display_name: Internet-Check
15:55:20 [Plugins] Executing: python3 /app/front/plugins/internet_ip/script.py prev_ip={prev_ip} INTRNT_DIG_GET_IP_ARG={INTRNT_DIG_GET_IP_ARG}
15:55:20 [INTRNT] In script
15:55:20 [INTRNT] INTRNT_DIG_GET_IP_ARG: -4 myip.opendns.com @resolver1.opendns.com
15:55:20 [INTRNT] - Retrieving Internet IP
15:55:20 [INTRNT]  DIG result : 95.160.111.16

15:55:20 [INTRNT]  Current internet_IP : 95.160.111.16
15:55:20 [INTRNT]          previous_IP : 95.160.111.16
15:55:20 [INTRNT] Finished 
15:55:20 [Plugins] SUCCESS, received 1 entries
15:55:20 [API] Updating table_appevents.json file in /front/api
15:55:20 [API] Updating table_plugins_history.json file in /front/api
15:55:20 [API] Updating table_plugins_objects.json file in /front/api
15:55:20 [Plugin utils] ---------------------------------------------
15:55:20 [Plugin utils] display_name: Arp-Scan (Network scan)
15:55:20 [Plugins] Executing: python3 /app/front/plugins/arp_scan/script.py userSubnets={subnets}
15:55:21 [ARP Scan] In script 
15:55:21 [ARP Scan] values.userSubnets: ["userSubnets=b'MTkyLjE2OC4xLjAvMjQgLS1pbnRlcmZhY2U9ZXRoMQ=='"]
15:55:21 [ARP Scan] userSubnetsParamBase64: 'MTkyLjE2OC4xLjAvMjQgLS1pbnRlcmZhY2U9ZXRoMQ=='
15:55:21 [ARP Scan] userSubnetsParam: 192.168.1.0/24 --interface=eth1
15:55:21 [Database] Opening DB
15:55:21 [ARP Scan] arpscan_output: 
15:55:21 [ARP Scan] Found: Devices without duplicates 0
15:55:21 Devices List len:0
15:55:21 Devices List:[]
15:55:21 [Plugins] No output received from the plugin ARPSCAN - enable LOG_LEVEL=debug and check logs
15:55:21 [Process Scan]  Processing scan results
15:55:21 [Process Scan] Print Stats
15:55:21 [Scan Stats] Devices Detected.......: 2
15:55:21 [Scan Stats] New Devices............: 0
15:55:21 [Scan Stats] Down Alerts............: 0
15:55:21 [Scan Stats] New Down Alerts........: 0
15:55:21 [Scan Stats] New Connections........: 0
15:55:21 [Scan Stats] Disconnections.........: 0
15:55:21 [Scan Stats] IP Changes.............: 0
15:55:21 [Scan Stats] Scan Method Statistics:
15:55:21     INTRNT: 1
15:55:21     local_MAC: 1
15:55:21 [Process Scan] Stats end
15:55:21 [Process Scan] Sessions Events (connect / discconnect)
15:55:21 [Process Scan] Creating new devices
15:55:21 [Process Scan] Updating Devices Info
15:55:21 [Process Scan] Voiding false (ghost) disconnections
15:55:21 [Process Scan] Pairing session events (connection / disconnection) 
15:55:21 [Process Scan] Creating sessions snapshot
15:55:21 [Process Scan] Inserting scan results into Online_History
15:55:21 [Process Scan] Skipping repeated notifications
15:55:21 [Skip Repeated Notifications] Skip Repeated
15:55:21 [Plugin utils] ---------------------------------------------
15:55:21 [Plugin utils] display_name: NSLOOKUP (Name discovery)
15:55:21 [Plugins] Executing: python3 /app/front/plugins/nslookup_scan/nslookup.py
15:55:21 [NSLOOKUP] In script
15:55:21 [Database] Opening DB
15:55:21 [NSLOOKUP] Unknown devices count: 0
15:55:21 [NSLOOKUP] Script finished
15:55:21 [Plugins] No output received from the plugin NSLOOKUP - enable LOG_LEVEL=debug and check logs
15:55:21 [Notification] Check if something to report
15:55:21 [Notification] Included sections: ['new_devices', 'down_devices', 'events']
15:55:21 [Notification] No changes to report
15:55:21 [MAIN] Process: Wait
15:55:26 [API] Updating table_appevents.json file in /front/api

### Debug enabled

- [X] I have read and followed the steps in the wiki link above and provided the required debug logs and the log section covers the time when the issue occurs.

**Comment by jokob-sk** [comment]
Hi there,

Could you please refresh the cache/try a different browser and if the issue persists check the developer console in the browser and share any errors that are logged? Here is an example guide where you can find the Dev console:

https://github.com/jokob-sk/NetAlertX/blob/main/docs/DEBUG_INVALID_JSON.md

**Comment by Savanta** [comment]
For some reason it started working by itself after a day.
Closing.

**Comment by crltc** [comment]
I am experiencing the same issue on a fresh install of 24.5.9 where the save button in the settings menu does not work. Multiple different browsers on multiple devices has not fixed it.

**Comment by jokob-sk** [comment]
Hi there, 

Could you please follow the advice mentioned here and report back? https://github.com/jokob-sk/NetAlertX/issues/636#issuecomment-2065328793

**Comment by crltc** [comment]
> Hi there,
> 
> Could you please follow the advice mentioned here and report back? [#636 (comment)](https://github.com/jokob-sk/NetAlertX/issues/636#issuecomment-2065328793)

For some reason I thought that I mentioned this in my original message but apparently didn't. I don't get any error response, all I get is the table_settings.json where the response is just my settings in json form.

**Comment by jokob-sk** [comment]
I think this issue is different from the original poster's one - can you open a new issue with additional information so I know what your debug logs say etc? 

Please include screenshots from your browser console. 

Thanks in advance,
j

**Comment by crltc** [comment]
> I think this issue is different from the original poster's one - can you open a new issue with additional information so I know what your debug logs say etc?
> 
> Please include screenshots from your browser console.
> 
> Thanks in advance, j

I redeployed it via Docker on Ubuntu 24.04 and it works as expected now, saving settings in the UI now works so I don't see a reason to open a new issue. Previously when I deployed with Docker on Debian 12 it wouldn't save on any desktop device/browser but for some reason it did work on mobile browsers. Just some data points for anyone who stumbles upon this in the future. 

Appreciate your quick response and willingness to help :) it's a great app!
